### ISSUE TITLE
Test for correctness: learning age and sex fertility probabilities [REAL DATA]

### ISSUE THREAD

**Issue by varunsatish** [varunsatish/llama-recipes-fertility#26]
**Testing process**
Training data: age x sex pairs, from real data, with fertility outcome. 

Fine-tune model, then conduct inference to obtain predicted probabilities for each age x sex pair (there should be ~ 90). 

Compare predicted probabilities to actual probabilities calculated from real data.

**Output**
Create plots, separately for different amounts of training data: 
- y axis: probability
- x axis: age 
- points: predicted vs real 
- facet: sex

**Comment by varunsatish** [comment]
@msalganik I have separate issues for real and synthetic data. Could you please have a skim and let me know if they sound right to you. The only difference, I believe, is the data.
